<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newr</class>
 <widget class="QMainWindow" name="newr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registro nuevo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">
QLineEdit{
    background-color: rgb(22,22,26);
    color: white;
    font: 12pt &quot;SansSerif&quot;;
    border-radius:10px;
    border: 2px solid rgb(80, 75, 68);
    padding: 5px;
}

QLineEdit:hover{
    background-color: rgb(22,22,26);
    border: 2px solid rgb(150,140,130);
}

QLineEdit:focus{
    background-color: rgb(1,1,1);
    border: 2px solid rgb(127, 90, 240);
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QTextEdit" name="txt_proyecto">
         <property name="placeholderText">
          <string>Proyecto</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTextEdit" name="txt_loop">
         <property name="placeholderText">
          <string>Ciclo</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTextEdit" name="txt_id">
         <property name="placeholderText">
          <string>ID caso de prueba</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QTextEdit" name="txt_caset">
         <property name="placeholderText">
          <string>Caso de prueba</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QTextEdit" name="txt_description">
         <property name="placeholderText">
          <string>Descripcion</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QTextEdit" name="txt_sources">
         <property name="placeholderText">
          <string>Pre requisitos</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QTextEdit" name="txt_datat">
         <property name="placeholderText">
          <string>Datos de prueba</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QTextEdit" name="txt_result">
         <property name="placeholderText">
          <string>Resultado esperado</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QTextEdit" name="txt_tester">
         <property name="placeholderText">
          <string>Tester</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #D52424;
    color: white;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #B71C1C;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btn_start">
         <property name="text">
          <string>Iniciar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_clean">
         <property name="text">
          <string>Limpiar datos</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_cancel">
         <property name="text">
          <string>Cancelar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txt_proyecto</tabstop>
  <tabstop>txt_loop</tabstop>
  <tabstop>txt_id</tabstop>
  <tabstop>txt_caset</tabstop>
  <tabstop>txt_description</tabstop>
  <tabstop>txt_sources</tabstop>
  <tabstop>txt_datat</tabstop>
  <tabstop>txt_result</tabstop>
  <tabstop>txt_tester</tabstop>
  <tabstop>btn_start</tabstop>
  <tabstop>btn_clean</tabstop>
  <tabstop>btn_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
